t3/api/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2018-2022 MobileCoin, Inc.
+
+//! T3 gRPC API.
+
+use mc_util_uri::{Uri, UriScheme};
+
+mod autogenerated_code {
+    pub use protobuf::well_known_types::Empty;
+
+    // Needed due to how to the auto-generated code references the Empty message.
+    pub mod empty {
+        pub use protobuf::well_known_types::Empty;
+    }
+
+    // Include the auto-generated code.
+    include!(concat!(env!("OUT_DIR"), "/protos-auto-gen/mod.rs"));
+}
+
+pub use autogenerated_code::{t3::*, *};
+
+pub type T3Uri = Uri<T3Scheme>;
+
+/// T3 Uri Scheme
+#[derive(Debug, Hash, Ord, PartialOrd, Eq, PartialEq, Clone)]
+pub struct T3Scheme {}
+impl UriScheme for T3Scheme {
+    /// The part before the '://' of a URL.
+    const SCHEME_SECURE: &'static str = "t3";
+    const SCHEME_INSECURE: &'static str = "insecure-t3";
+
+    /// Default port numbers
+    const DEFAULT_SECURE_PORT: u16 = 5553;
+    const DEFAULT_INSECURE_PORT: u16 = 5554;
```

Comment:
IIUC these should be the HTTP/S ports
```suggestion
    const DEFAULT_SECURE_PORT: u16 = 443;
    const DEFAULT_INSECURE_PORT: u16 = 80;
```

Reply:
I will check the T3 docs for this

Reply:
I have a feeling @remoun is right. I wouldn't count on the docs, I would look to see which port Moby is connecting to